lua/nvim-navic/init.lua

```diff
@@ -403,8 +403,9 @@ function M.setup(opts)
 end
 
 -- returns table of context or nil
-function M.get_data()
-	local context_data = navic_context_data[vim.api.nvim_get_current_buf()]
+function M.get_data(bufnr)
+	bufnr = bufnr or vim.api.nvim_get_current_buf()
```

Comment:
bufnr should be local variable.

Reply:
It already is a local variable as it was passed through function parameters.